explorer/lib/etherUnits.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+var BigNumber = require('bignumber.js');
+
+var etherUnits = function() {}
+etherUnits.unitMap = {
+	'wei': '1',
+	'kwei': '1000',
+	'ada': '1000',
+	'femtoether': '1000',
+	'mwei': '1000000',
+	'babbage': '1000000',
+	'picoether': '1000000',
+	'gwei': '1000000000',
+	'shannon': '1000000000',
+	'nanoether': '1000000000',
+	'nano': '1000000000',
+	'szabo': '1000000000000',
+	'microether': '1000000000000',
+	'micro': '1000000000000',
+	'finney': '1000000000000000',
+	'milliether': '1000000000000000',
+	'milli': '1000000000000000',
+	'ether': '1000000000000000000',

```

Comment:
We need to create an issue for these fields.

Reply:
what sort of issue?